bin/stacks/routing-api-stack.ts

```diff
@@ -259,22 +261,123 @@ export class RoutingAPIStack extends cdk.Stack {
         period: Duration.minutes(5),
         statistic: 'p90',
       }),
-      threshold: 7500,
+      threshold: 5500,
       evaluationPeriods: 3,
     })
 
     const simulationAlarmSev2 = new aws_cloudwatch.Alarm(this, 'RoutingAPI-SEV2-Simulation', {
       alarmName: 'RoutingAPI-SEV2-Simulation',
-      metric: api.metric('SimulationFailed'),
-      threshold: 3,
-      evaluationPeriods: 3,
+      metric: new MathExpression({
+        expression: '100*(simulationFailed/simulationRequested)',
+        usingMetrics: {
+          simulationRequested: new aws_cloudwatch.Metric({
+            namespace: 'Uniswap',
```

Comment:
does it make sense to use a more specific namespace?

Reply:
yes, but it is a bit more complicated change than you think. Here, it is not actually creating the metric, it is just retrieving the metric from the namespace. The namespace is actually set here: https://github.com/Uniswap/routing-api/blob/88eb9bef1f3e6f164791414ba2ae0dfc5ad7a295/lib/handlers/quote/injector.ts#L54
And the same namespace is used for all the custom metrics in routing-api. Good idea to split up the namespace, but it should be a separate PR imo